src/Microsoft.ML.EntryPoints/MacroUtils.cs

```diff
@@ -22,7 +22,7 @@ public enum TrainerKinds
         {
             SignatureBinaryClassifierTrainer,
             SignatureMultiClassClassifierTrainer,
-            SignatureRankerTrainer,
+            SignatureRankingTrainer,
```

Comment:
>SignatureRankingTrainer [](start = 12, length = 23)

Another thing where this is I think hypercorrected. See how it is now inconsistent with everything else here?